source/perflib/basename.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifdef	_WIN32
+
+#include <string.h>
+
+/*
+ * windows variant of basename(3). works on ASCIIZ only.
+ * simple and perhaps naive implementation too.
+ */
+const char *
+basename(const char *path)
```

Comment:
nit: Why is this put on a new line?

Reply:
sorry about it, this is a *BSD/Solaris coding style which I'm used to. will fix it.

FYI: the advantage of this style is you can quickly find a function definition by matching `^function` regex.